nodejs/CMakeLists.txt

```diff
@@ -29,16 +40,17 @@ set_target_properties(onnxruntime_binding PROPERTIES
   INSTALL_RPATH_USE_LINK_PATH FALSE)
 target_link_libraries(onnxruntime_binding PRIVATE ${CMAKE_JS_LIB})
 
+# add libraries
 if (WIN32)
   target_link_libraries(onnxruntime_binding PRIVATE
-    ${CMAKE_SOURCE_DIR}/../build/Windows/${CMAKE_BUILD_TYPE}/${CMAKE_BUILD_TYPE}/onnxruntime.lib)
+    ${ONNXRUNTIME_BUILD_DIR}/${CMAKE_BUILD_TYPE}/onnxruntime.lib)
 elseif (APPLE)
   target_link_libraries(onnxruntime_binding PRIVATE
-    ${CMAKE_SOURCE_DIR}/../build/Linux/${CMAKE_BUILD_TYPE}/libonnxruntime.${ort_version}.dylib)
+    ${ONNXRUNTIME_BUILD_DIR}/libonnxruntime.${ort_version}.dylib)
   set_target_properties(onnxruntime_binding PROPERTIES INSTALL_RPATH "@loader_path")
 else()
   target_link_libraries(onnxruntime_binding PRIVATE
-    ${CMAKE_SOURCE_DIR}/../build/Linux/${CMAKE_BUILD_TYPE}/libonnxruntime.so.${ort_version})
+    ${ONNXRUNTIME_BUILD_DIR}/libonnxruntime.so.${ort_version})
```

Comment:
Usually, on Linux people do it in a way of:
"-L${ONNXRUNTIME_BUILD_DIR} -lonnxruntime"

Because if you put a full path there, the full path name will be put into the final onnxruntime_binding.so, and when you copy onnxruntime_binding.so to another machine, it may not work because ONNXRUNTIME_BUILD_DIR doesn't exist anymore.

Reply:
used `target_link_directories` to specify link search folder